.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: cargo
```

Comment:
Do we need an indent here? As https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#example-of-a-basic-setup-for-three-package-managers shows, the whole `package-ecosystem` object should be one of the list of values ​​of `updates`.

Reply:
Fixed now. And I also added the go dependency, thanks for catching this @Xynnn007

Reply:
I see in that link that there's also tracking for github actions:

```yaml
  # Maintain dependencies for GitHub Actions
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "weekly"
```

should we add that as well?

Reply:
Good suggestion, done now. Thanks.